src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml

```diff
@@ -25,20 +25,39 @@
             <h2 id="scheme-details-h2-responsibility" class="govuk-heading-m"> Employer responsibilities </h2>
             @Model.Scheme.Responsibility
             <h2 id="scheme-details-h2-benefits" class="govuk-heading-m">Benefits of @Model.Scheme.Name</h2>
-            @Model.Scheme.Benefits
-            @if (@Model.Scheme.CaseStudies != null)
+                @Model.Scheme.Benefits
```

Comment:
Very minor but looks like the indentation of this line has been knocked out.

Reply:
Fixed in latest commit